linkerd/examples/admin-tls.yaml

```diff
@@ -0,0 +1,48 @@
+# Config for serving linkerd admin page over TLS
+# localhost:9990 and localhost:4140/foo/ should display the same page.
```

Comment:
I think the comments on lines 2-5 are not really relevant to this example and can be removed.